eslint.config.js

```
@@ -27,7 +27,7 @@ export default tseslint.config(
     rules: {
       ...reactHooks.configs.recommended.rules,
       'no-console': ['warn'],
-      'prettier/prettier': 'error',
+      // 'prettier/prettier': 'error',
```

Why is this commented? It should throw an error if prettier is not working.

---

it was throwing errors which were caused by prettier formatter, like dangling commas after last element